src/main/java/Content.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * @{code ContentInput}, providing read operation, and @{code ContentOutput}, providing write operation
```

Comment:
@ArthurRomasyuk The documentation is still weak. In fact you're saying more about `ContentInput` and `ContentOutput` instead of providing documentation for `Content`. What is that class about? Why would I need it?